plugins/chat/assets/javascripts/discourse/components/styleguide/chat-header-icon.hbs

```diff
@@ -0,0 +1,59 @@
+<StyleguideExample @title="<Chat::Header::Icon>">
+  <Styleguide::Component>
+    <header
+      class="d-header"
+      style="display: flex; align-items: center; justify-content: center;"
+    >
```

Comment:
I know we don't generally do this but I think it makes sense here to 1) mimic expected placement in the app 2) have everything located in this file